testing/TestHarness/TestHarness/BaseHostInitialization.cs

```diff
@@ -7,7 +7,7 @@ public abstract class BaseHostInitialization : IHostInitialization
 	public virtual IHost InitializeHost(Window window)
 	{
 		var host = UnoHost
-				.CreateDefaultBuilder()
+				.CreateDefaultBuilder(Assembly.GetEntryAssembly())
```

Comment:
While this is for testing, this won't work on Android with MonoVM and CoreCLR, as `Assembly.GetEntryAssembly()` returns `null` there.

Perhaps this code won't ever run on Android?  But if it *does*, then will this work as expected/intended?